test/valid/validate_test.cc

```diff
@@ -208,6 +208,25 @@ TEST(ValidateTest, ConstantExpression_WrongInstructionCount) {
                         VT_I32, 0));
 }
 
+TEST(ValidateTest, ConstantExpression_GC) {
```

Comment:
Is what you call `ConstantExpression` here what I normally think of an *initializer expression*?   Why choose a different term here?

Reply:
`ConstantExpression` is what the spec calls it: https://webassembly.github.io/spec/core/valid/instructions.html#constant-expressions

They're used for global initializers but also for data/element segments.